src/components/pages/index.tsx

```diff
@@ -41,6 +41,11 @@ const pageRoutes = [
     component: withSuspense('Dashboard'),
     exact: true,
   },
+  {
+    path: '/editform',
+    component: withSuspense('MobileForm'),
```

Comment:
MobileForm not found

Reply:
I used it for testing. I just forgot to remove it.